OperatorProblem3_latest.js

```diff
@@ -0,0 +1,62 @@
+// ***********  JavaScript Task #2 Operator Problem  **********************
+const array = [
+  "OR",
+  ["<", "a", "b"],
+  ["AND", ["==", "c", "d"], ["!=", "e", "f"]],
+];
+const newArray = [];
+const infixExpression = recursiveFunction(array); // passing given array to recursiveFunction function
```

Comment:
There are too many comments here and there.
Only write comments when the function is complex. The comments should tell why this is complex instead of its implementation detail (how it works)

Reply:
Noted Ahmed bhai